python/tank/templatekey.py

```diff
@@ -17,8 +17,12 @@
 from . import constants
 from .errors import TankError
 from .util import sgre as re
-from tank_vendor import six
-from tank_vendor.six.moves import zip
+from tank_vendor.six.moves import zip, collections_abc
```

Comment:
Why do we keep these? Can we import the collections.abc module instead (added in Python 3.3)? 

And for zip, I think it was changed even earlier.

Reply:
@eduardoChaucaGallegos please answer this point.

Reply:
It's going to be part of phase 3 of plan to remove python 2

Reply:
Hum, that does not seem right when comparing to this comment: https://github.com/shotgunsoftware/tk-desktop2/pull/43#discussion_r1683816491

Reply:
in fact, I put the same here https://github.com/shotgunsoftware/tk-desktop2/pull/43#discussion_r1700473879